index.src.html

```diff
@@ -17,96 +17,99 @@
 module containing the credential is functioning as an embedded authenticator. However, the credential may also be accessed over
 BLE by a user agent on a nearby laptop. In this latter case, the phone is functioning as an external authenticator. These modes
-may even be used in a single end-to-end user scenario. One such scenario is described in the remainder of this section.
+may even be used in a single end-to-end user scenario. One such scenario is described among the use cases below (in [[#authentication-external]]).
+
 
+## Use Cases ## {#use-cases}
 
-## Registration (<a>embedded authenticator</a> mode) ## {#registration-embedded}
+The below use case scenarios illustrate both <a>embedded authenticator</a> and <a>external authenticator</a> modes, as well as outline further scenarios. 
 
-- On the phone:
-    * User goes to example.com in the browser and signs in to an existing account using whatever method they have been using
+### Registration (<a>embedded authenticator</a> mode) ### {#registration-embedded}
+
+- On a phone:
+    * User navigates to example.com in a browser and signs in to an existing account using whatever method they have been using
         (possibly a legacy method such as a password), or creates a new account.
     * The phone prompts, "Do you want to register this device with example.com?"
     * User agrees.
-    * The phone prompts the user for a previously configured authorization gesture (PIN, biometric, etc.); the user provides
+    * The phone prompts the user for a previously configured <a>authorization gesture</a> (PIN, biometric, etc.); the user provides
         this.
     * Website shows message, "Registration complete."
 
 
-## Authentication (<a>external authenticator</a> mode) ## {#authentication-external}
+### Authentication (<a>external authenticator</a> mode) ### {#authentication-external}
 
-- On the laptop:
-    * User goes to example.com in browser, sees an option "Sign in with your phone."
+- On a laptop:
+    * User navigates to example.com in a browser, sees an option to "Sign in with your phone."
     * User chooses this option and gets a message from the browser, "Please complete this action on your phone."
 
-- Next, on the phone:
+- Next, on their phone:
     * User sees a discreet prompt or notification, "Sign in to example.com."
     * User selects this prompt / notification.
     * User is shown a list of their example.com identities, e.g., "Sign in as Alice / Sign in as Bob."
-    * User picks an identity, is prompted for an authorization gesture (PIN, biometric, etc.) and provides this.
+    * User picks an identity, is prompted for an <a>authorization gesture</a> (PIN, biometric, etc.) and provides this.
 
-- Now, on the laptop:
-    * Web page shows that the selected user is signed in, and navigates to the signed-in page.
+- Now, back on the laptop:
+    * Web page shows that the selected user is signed-in, and navigates to the signed-in page.
```

Comment:
oh gosh, I don't really care that much, but it seems to me that in the present cited sentence (L90), the first instance of "signed in"/"signed-in" is not a verb but a noun as it is being used to summarize/name the user's present state of being signed-in:

  `Web page shows that the selected user is signed-in, ...`

..rather than saying something like..

 `The user was signed in by the web app...`
 `The user signs in to the web app..."